docs/install-with-kind-k8s.md

```diff
@@ -0,0 +1,93 @@
+- create kind cluster
+
+```
+cat <<EOF | kind create cluster --name mine --config=-
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+- role: control-plane
+  kubeadmConfigPatches:
+  - |
+    kind: InitConfiguration
+    nodeRegistration:
+      kubeletExtraArgs:
+        node-labels: "ingress-ready=true"
+        authorization-mode: "AlwaysAllow"
+  extraPortMappings:
+  - containerPort: 80
+    hostPort: 80
+    protocol: TCP
+  - containerPort: 443
+    hostPort: 443
+    protocol: TCP
+- role: worker
+- role: worker
+- role: worker
+EOF
+```
+
+- install go  gofmt  golangci-lint  kube-apiserver  kubebuilder  kubectl  kustomize  pkger etcd
```

Comment:
@bitsf 

Could you please list the related links of installation documents of such tools?

Reply:
`make dev-tools` not fully work yet, so add some workaround.
You can help check if this workaround can be add to the makefile too